### DISCUSSION TITLE
How to make unique design?

### DISCUSSION THREAD

**Discussion by MonMohon** [callstack/react-native-paper#4533] [Q&A]
I can't able to figure out how to make it unique design apply material design.
https://callstack.github.io/react-native-paper/docs/guides/theming#creating-dynamic-theme-colors
Two app with same theme its 100% clone.
Please Help

**Comment by gedu** [comment]
Hey, are you looking to create a theme with your colors and use it in two different apps?

I would create `theme` object with the object that you can find in the link you post, and change the colors, you then export it and you can add it into `<PaperProvider theme={yourTheme}>`

**Reply by MonMohon** [reply]
ok, I got it now, I just want to clarify this like onSecondary, secondaryContainer etc

how to use it.

and one last thing is it necessary to merge with MD3Theme
const theme = {
  ...MD3LightTheme,
  colors: yourGeneratedLightOrDarkScheme.colors, // Copy it from the color codes scheme and then use it here
};

but after merge I didn't get the custom theme color

thank you

**Reply by MonMohon** [reply]
import { MD3Colors } from 'react-native-paper';
MD3Colors.error50

this also help if I get all the MD3Colors

**Reply by gedu** [reply]
If you will override all the fields, you don’t need to import the defaults, but if you want to keep some, then yes, maybe you want to keep the error colors. Probably [this will help](https://m3.material.io/styles/color/system/how-the-system-works) you with primary, onPrimary, etc. We are planning to abstract that part of the theme to make it more developer experience (DX) friendly.

**Reply by MonMohon** [reply]
thanks you so much

**Reply by MonMohon** [reply]
yes, that perfect first I've to know the MD3 system

**Comment by MonMohon** [comment]
![1000112305](https://github.com/user-attachments/assets/a017cfdc-5778-48a0-8e34-43d26dd7bf30)

how to disable active bar icon background color

**Reply by MonMohon** [reply]
react-native-paper/react-navigation
